### DISCUSSION TITLE
Android Version 8.1

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#146] [General]
from which version of android does the application work ?

**Comment by textbrowser** [comment]
I have tried it on a tablet using Android 8.1. Smaller devices will work too in landscape mode. I cannot test all versions of Android. If it works in 8.1, it should be functional in newer versions.

**Reply by meteos77** [reply]
I am in 4.4.2 :(

**Reply by textbrowser** [reply]
I don't have anything that old. Is it end-of-life?

**Reply by meteos77** [reply]
not for my use
not internet , kiwix , osmand, card game , puzzle, photo gallery dictionaries

**Reply by meteos77** [reply]
Test install BQ  : Analysis error  : a problem occurred during the analysis of the package.  damage

**Reply by textbrowser** [reply]
Yah, I'm not surprised. Qt has minimum requirements too. :)

**Reply by meteos77** [reply]
I don't like android very much the more it advances in version the less simple it is

**Comment by textbrowser** [comment]
https://doc.qt.io/qt-5/android.html

Android 5 and newer. :)

**Reply by meteos77** [reply]
as a user what does qt6 bring compared to qt5 ?

**Reply by meteos77** [reply]
my tablet will accept android 5 in my to do list :-)

**Reply by textbrowser** [reply]
See https://doc.qt.io/qt-6/whatsnew62.html.

**Reply by textbrowser** [reply]
Qt's data visualization package is becoming impressive. QML can be used to create beautiful displays for sensors.
